scripts/install-prereqs

```diff
@@ -77,3 +77,11 @@ EOF
 
     apt-get -y install clang-7
 fi
+
+# Install cmake 3.13.1
```

Comment:
Could we change this just slightly to do the following:

if (cmake is not present at all) then (install via download)
if (cmake is installed but not <= 3.13.1) then (exit with an error message)

That way we only install if the user hadn't already installed it somehow, but we still check that it installed correctly (regardless of how it was installed).

Thank you!

Reply:
Changes added!